packages/dynamite/dynamite/lib/src/models/type_result/base.dart

```diff
@@ -30,7 +30,11 @@ class TypeResultBase extends TypeResult {
           return '$object.toString()';
         }
       case 'application/octet-stream':
-        return 'utf8.encode($object) as Uint8List';
+        if (className == 'Uint8List') {
+          return object;
+        } else {
+          return 'utf8.encode($object.toString()) as Uint8List';
```

Comment:
I don't think this makes sense.
I'd rather add an elseif to check whether the type is a string and throw otherwise.
The `toString` method can be somewhat unpredictable.